interface/themes/core.scss

```diff
@@ -9,7 +9,7 @@
  */
 
 // Import Select2 BS4 Theme
-@import "../../public/assets/@ttskch/select2-bootstrap4-theme/src/layout";
+@import "../../public/assets/@ttskch/select2-bootstrap4-theme/src/select2-bootstrap4";
```

Comment:
@bradymiller This seems to have fixed the issue although I've rebuilt several times in past with no joy till tried again this a.m and works so, beats me. Maybe a windows thing.
I'm going to revert config change and style the selection remove 'x' so easier for mobile devices.
Then I guess we'll see on demos.